src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -315,7 +315,7 @@
   "skyblocker.config.helpers.bazaar.enableBazaarHelper.@Tooltip": "Draws icons on top of orders to explain the current state of the order.\n\n%s: Order is going to expire soon\n%s: Order has expired\n%s: Order is filled to some degree and there are items/coins to claim\n%s: Order is filled",
   "skyblocker.config.helpers.bazaar.enableBazaarLookup": "Enable Bazaar Lookup",
   "skyblocker.config.helpers.bazaar.enableBazaarLookup.@Tooltip": "Searches the Bazaar for the current hovered item with the F6 key.",
```

Comment:
update these 2 lines to inform that it searches bazaar. Add to the tooltip that you need cookie buff